### PR TITLE
Pr 178/mobile navbar

### PR DESCRIPTION
**Pull request by hdJerry**
- [x] Develop navbar (hamburgher icon and menu)
- [x] Make sure if follows the figma designs
- [x] Make if accessible with the keyboard

Screenshot from 2022-04-22 14-38-11

Screenshot from 2022-04-22 14-37-26

### CODE REVIEW

**Review by Zelig880 — CHANGES_REQUESTED** [review]
The storybook story has not been updated as requested by the Story. Please update

**Comment by hdJerry** [comment]
> The storybook story has not been updated as requested by the Story. Please update

The Story book for the navbar itself will be for this and this is the responsive of the navbar except we are to create its as a component of its own.

File: src/components/NavBar/NavBar.jsx [hunk 1]
```diff
@@ -27,24 +30,35 @@ const links = [
...
+  const [showNavLinks, setShowNavLinks] = useState(false);
   const { pathname } = useRouter();
   const isActive = (path, to) => {
     if (path.split('/')[1] === to.split('/')[1]) {
       return true;
     }
     return false;
   };
+  const toggleMenuLink = (value) => {
+    document.body.style.overflow = value ? 'hidden' : 'auto';
```
Comment (markshenouda) [inline src/components/NavBar/NavBar.jsx:42]: As Dustin said before accessing dom is expensive in React! We should find another way to do it

File: src/components/NavBar/NavBar.module.css [hunk 2]
```diff
@@ -5,34 +5,46 @@
 .container {
   margin-inline: auto;
   max-width: 1440px;
-  padding: 1.5rem;
+  padding: 1.2rem 1rem;
   position: relative;
 }
```
Comment (markshenouda) [inline src/components/NavBar/NavBar.module.css:10]: I'd recommend adding a z-index here so you can avoid this small bug


https://user-images.githubusercontent.com/49324987/165765309-40bc70a7-99f0-4741-861b-ac7479388bc2.mov


https://user-images.githubusercontent.com/49324987/165765309-40bc70a7-99f0-4741-861b-ac7479388bc2.mov



https://user-images.githubusercontent.com/49324987/165765149-c0ae574f-d54c-4acc-8ced-8b8e2e0c4f54.mov

Comment (markshenouda) [inline src/components/NavBar/NavBar.module.css:10]: I'd recommend adding a z-index here so you can avoid this small bug


https://user-images.githubusercontent.com/49324987/165765309-40bc70a7-99f0-4741-861b-ac7479388bc2.mov


https://user-images.githubusercontent.com/49324987/165765309-40bc70a7-99f0-4741-861b-ac7479388bc2.mov



https://user-images.githubusercontent.com/49324987/165765149-c0ae574f-d54c-4acc-8ced-8b8e2e0c4f54.mov

**Review by markshenouda — CHANGES_REQUESTED** [review]

**Review by markshenouda — APPROVED** [review]

**Pull request merged**